plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.tsx

```diff
@@ -176,21 +174,7 @@ function ProductCard( props: ProductCardProps ): JSX.Element {
 				) }
 				<div className="woocommerce-marketplace__product-card__price">
 					{ ! isLoading && (
-						<>
-							<span className="woocommerce-marketplace__product-card__price-label">
-								{
-									// '0' is a free product
-									product.price === 0
-										? __( 'Free download', 'woocommerce' )
-										: currencySymbol + product.price
-								}
-							</span>
-							<span className="woocommerce-marketplace__product-card__price-billing">
-								{ product.price === 0
-									? ''
-									: __( ' annually', 'woocommerce' ) }
-							</span>
-						</>
+						<ProductPrice product={ props.product } />
```

Comment:
@kdevnel also did a similar refactor and he named it `ProductFooter` (or something like that). I think that's a better approach. I'll update this to match what Kyle did.